package.json

```diff
@@ -17,9 +17,9 @@
   },
   "dependencies": {
     "autoprefixer": "7.2.4",
-    "babel-core": "6.26.0",
-    "babel-loader": "7.1.4",
-    "babel-preset-env": "1.6.1",
+    "babel-core": "7.0.0-beta.3",
```

Comment:
Since babel is a build pipeline tool, only its products are what we'd need to be concerned about. If it passes at the build stage we can be reasonably certain it's fine to use, at least in terms of our build processes. I guess checkmarx would flag any issues on the output (if it checks production bundles—checking with @lfatty).

For what it's worth, beta 3 has been [out for a year](https://github.com/babel/babel/blob/master/packages/babylon/CHANGELOG.md#700-beta3-2017-02-28).

If we didn't want to use the beta version at this time, I could manually add the browser list to the analytics-gtm webpack config for now, and keep this PR as a reference for updating to a more automated workflow in the future.

Reply:
Sounds fine to me, just want to make sure we avoid the repeat of when I pushed us to upgrade Node a bit prematurely a year ago and it was a world of pain.